### DISCUSSION TITLE
how to suppress branches with conflicts from `jj status`?

### DISCUSSION THREAD

**Discussion by wez** [jj-vcs/jj#3035] [Q&A]
or perhaps phrased differently: is there a way to show only the status of files in the working copy?

My status output is dominated by information about branches that, at this moment in time, I do not care about:

```console
$ jj st
The working copy is clean
Working copy : mkqwvnwm ac5fb4d9 (empty) (no description set)
Parent commit: nyxtnmyl 22f9f8d2 main?? main@git main@origin | docs: changelog for #5001
These branches have conflicts:
  0.17-smithay
  centos7
  feature/add-blur-on-macos
  ime
  main
  scrollback
  svg
  x11resizecoalesce
  Use `jj branch list` to see details. Use `jj branch set <name> -r <rev>` to resolve.
```

Part of my workflow is to frequently (perhaps due to some OCD) check what files I've changed. In the current state, nothing has changed but I have to work very hard to discern that here, partly because the status output is new an unfamiliar to me at the moment, but largely because there is so much other information that is not relevant to my goal in that moment.

Is there a way I can constrain this output, or a different command I should use if I want just the working copy state and get something like:

```console
$ jj SOMETHING
The working copy is clean
```

**Comment by wez** [comment]
Hmm, maybe `jj diff --summary` is what I should compulsively use?  Are there other consequences to that? What other important information am I likely to miss if I make my `st` alias run that instead of `jj status`?

**Comment by martinvonz** [comment]
You shouldn't normally have conflicted branches. Maybe you have multiple remotes where the branches have diverged relative to each other? The default these days is to not import branches, which avoids running into conflicts like that. See
https://martinvonz.github.io/jj/v0.14.0/config/#automatic-local-branch-creation for how to stop tracking branches that you're already tracking

> or a different command I should use if I want just the working copy state

Maybe `jj diff -s` (as you discovered yourself)

**Reply by wez** [reply]
I did have `git.auto-local-branch` enabled because it sounded like I should have it in order to be able to see the branches that already exist in my locally imported git repo. I've turned that off and re-imported the repo to jj.

Please forgive my ignorance here; I don't really understand why I have to consider all those other branch conflicts on each status call.

**Reply by martinvonz** [reply]
The thinking is that you would want to know and resolve the divergent branches pretty soon.
